abis/linux/shm.h

```diff
@@ -59,10 +59,7 @@ struct shmid_ds {
 	pid_t shm_cpid;
 	pid_t shm_lpid;
 	unsigned long shm_nattch;
-	unsigned long __unused[3];
-	time_t shm_atime;
-	time_t shm_dtime;
-	time_t shm_ctime;
+	unsigned long __unused[2];
 };
```

Comment:
This does not match glibc/musl. In particular, removing `shm_{a,d,c}time` without replacement will most likely break downstream programs.

Reply:
Fixed in the fixup commit - the layout already matches what glibc exposes, only changing the names was necessary.